keyboards/minidox/keymaps/djuggler/keymap.c

```diff
@@ -0,0 +1,200 @@
+#include "minidox.h"
+#include "action_layer.h"
+#include "eeconfig.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
```

Comment:
Lines 11-15 should be replaced with:

```c
enum layer_names {
	_QWERTY,
	_COLEMAK,
	_LOWER,
	_RAISE,
	_ADJUST,
};
```

Having the _RAISE layer at 3 and _ADJUST at 16 will make the compiler insert empty layers at each index from 4 to 15, increasing firmware size for no change in functionality.